frontend/app/(tabs)/create-recipe.tsx

```diff
@@ -1,92 +1,81 @@
-import { useRouter, Link, useLocalSearchParams } from "expo-router";
```

Comment:
Looks a lot better here, glad you split it up into two files.

Reply:
Thanks for the recommendation earlier!